extras/bampfa/app/assets/stylesheets/_variables.scss

```diff
@@ -6,3 +6,4 @@ $warning: #b8c924;
 $danger: #d62518;
 $light: lightgray;
 $dark: #343a40;
+$gray-900: #151519;
```

Comment:
This color is used for body text. I'm overriding it to make it a slightly darker near-black.